fmc/src/flow/pcr.rs

```diff
@@ -23,9 +23,17 @@ Note:
 use crate::flow::tci::Tci;
 use crate::fmc_env::FmcEnv;
 use crate::HandOff;
-use caliptra_drivers::{okref, CaliptraResult, PcrId};
+use caliptra_drivers::{
+    cprintln,
+    memory_layout::{PCR_LOG_ORG, PCR_LOG_SIZE},
+    okref,
+    pcr_log::{PcrLogEntry, PcrLogEntryId},
+    CaliptraResult, PcrBank, PcrId,
+};
 
 use caliptra_common::{RT_FW_CURRENT_PCR, RT_FW_JOURNEY_PCR};
+use caliptra_error::CaliptraError;
+use zerocopy::AsBytes;
 
 /// Extend current PCR
```

Comment:
This function erases the current PCR if the reset reason is an update reset. Can we have test coverage for that?